register_forte_options.py

```diff
@@ -90,7 +90,8 @@ def register_driver_options(options):
 
     options.add_int("PRINT", 1, "Set the print level.")
 
-    options.add_bool("READ_ORBITALS", True, "Read orbitals from file if true")
+    options.add_bool("READ_ORBITALS", False, "Read orbitals from file if true")
```

Comment:
Yes, it is in conflict with the QC branch, which is done in C++ side. The Python implementation uses PSI API and numpy. Afterall, Ca is a Matrix from Psi4.

I turned it off by default is because reading orbitals now occurs at the very beginning and it will print 'file NOT found' if 'forte_Ca.npz' file is not available. I just don't want this extra line in the output.